rmgpy/cantherm/statmech.py

```diff
@@ -318,6 +311,16 @@ def load(self):
         logging.debug('    Reading optimized geometry...')
         coordinates, number, mass = geomLog.loadGeometry()
 
+        # Infer atoms from geometry
+        atoms = {}
+        for atom_num in number:
+            symbol = atom_num_dict[atom_num]
```

Comment:
We should capture exception when `atom_num` is not in `atom_num_dic`, and output an error message that this atom is currently not supported in Cantherm

Reply:
Done. Since we're now allowing the user to input atomic energies, I also added more enthalpies of formation of atoms so that we now support more elements.